src/Components/ExtractLevelsMenu.jsx

```diff
@@ -0,0 +1,184 @@
+
+import React, {useState, useEffect} from 'react'
```

Comment:
Please read this and use the sorting alg:
https://github.com/bldrs-ai/Share/wiki/Dev:-Style

Reply:
Can you check if my fix is correct?